fuel/contributing_model.md

```diff
@@ -0,0 +1,8 @@
+# Contributing a New Model
+
+The easiest way to contribute a new model is via [https://app.gazebosim.org/fuel/models/upload](https://app.gazebosim.org/fuel/models/upload).
+
+Fuel assumes that a model conforms to Gazebo's model structure. At minimum,
```

Comment:
Is it useful/maintainable to link to this more detailed tutorial? https://gazebosim.org/api/sim/7/meshtofuel.html

Reply:
Thanks. Added in f1c6049b645e575f39a66c4eee2fb47884129079